contracts/VetoERC20Voting.sol

```diff
@@ -0,0 +1,181 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./interfaces/IERC20Votes.sol";
+import "./interfaces/IVetoGuard.sol";
+import "./interfaces/IVetoERC20Voting.sol";
+import "./TransactionHasher.sol";
+import "@fractal-framework/core-contracts/contracts/ModuleBase.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+/// @notice A contract for casting veto votes with an ERC20 votes token
+contract VetoERC20Voting is IVetoERC20Voting, TransactionHasher, ModuleBase {
```

Comment:
Yeah, don't think we need it. I included ModuleBase because of a discussion Adam and I had last week, to build this contract as a Fractal module for now, knowing that it will change in the future. 

I'm definitely good with removing ModuleBase here. 

@adamgall Thoughts?

Reply:
The existence of `ModuleBase` on the contract right now -- is it so that the UI can recognize it as a Fractal module?

I think that with how quickly we are navigating into this new Gnosis-centric model, we can probably remove any Fractal MVD-specific code for now.

@herbig I wonder if it makes sense to "pause" this code / feature / PR entirely for right now, until we have a better understanding of what our architectural approach for our Gnosis-centric model will be. At that point, we can revisit this PR and make sure it fits.

Reply:
@adamgall @tbwebb22 Yeah, we'll likely need to integrate whatever new contracts come out of the new paradigm, so pausing might be a good idea.

Reply:
I am not sure why we would need to pause at this moment - the code is already deployed so changes here will not affect the current code base - I would love to keep pushing on building out this feature.

Reply:
fair enough, this will still be needed, but perhaps restructured for the next phase of contracts

Reply:
Un-pausing now - I think we have a good enough idea of what our architecture may look like to get this PR merged in

Reply:
Ah yes I did intend to remove the ModuleBase related stuff based on a conversation we hard earlier this week. Just removed it in latest commit.